packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -498,6 +498,9 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
       return;
     if (!canDrag) {
       _gestureRecognizers = const <Type, GestureRecognizerFactory>{};
+      // Cancel active drag/hold activities because we're disposing the gesture
```

Comment:
nit: instead of "we" maybe we should be specific here and say that our RawGestureDetector is disposing them.